ray_image_preprocessing.py

```diff
@@ -0,0 +1,136 @@
+"""
+Preprocessing helper functions for the MAPLE pipeline.
+
+Project: Permafrost Discovery Gateway: Mapping Application for Arctic Permafrost Land Environment(MAPLE)
+PI      : Chandi Witharana
+"""
+
+
+import os
+from pathlib import Path
+import shutil
+from typing import Dict, Any
+
+import numpy as np
+from osgeo import gdal
+from skimage import filters, io
+from skimage.morphology import disk
+
+from mpl_config import MPL_Config
+import gdal_virtual_file_system as gdal_vfs
```

Comment:
nit: Usually it's nice to have the raw "imports" at the top and then following by the "from ... import ..."

Reply:
This applies to all files in the PR. It would also be great if we could make the effort to alphabetize the imports!

Reply:
hey - I was following the python style guide of:

Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

https://peps.python.org/pep-0008/#imports

I will try to make sure they're alphabetical too